scripts/fix-tslint.ts

```diff
@@ -2,44 +2,44 @@
 
 /// <reference types="node" />
 
-import * as fs from "fs";
-import * as path from "path";
-import JSON = require("comment-json");
+import * as fs from 'node:fs';
+import JSON from 'comment-json';
 
-const home = path.join(__dirname, "..", "types");
+const home = new URL('../types/', import.meta.url);
 
 for (const dirName of fs.readdirSync(home)) {
-    if (dirName.startsWith(".") || dirName === "node_modules" || dirName === "scripts") {
+    if (dirName.startsWith('.') || dirName === 'node_modules' || dirName === 'scripts') {
         continue;
     }
 
-    const dir = path.join(home, dirName);
+    const dir = new URL(`${dirName}/`, home);
     const stats = fs.lstatSync(dir);
     if (stats.isDirectory()) {
         fixTslint(dir);
         // Also do it for old versions
         for (const subdir of fs.readdirSync(dir)) {
             if (/^v\d+$/.test(subdir)) {
-                fixTslint(path.join(dir, subdir));
+                fixTslint(new URL(`${subdir}/`, dir));
             }
         }
     }
 }
 
-function fixTslint(dir: string): void {
-    const target = path.join(dir, 'tslint.json');
+function fixTslint(dir: URL): void {
+    const target = new URL('tslint.json', dir);
     if (!fs.existsSync(target)) return;
     let json = JSON.parse(fs.readFileSync(target, 'utf-8'));
     json = fix(json);
-    const text = Object.keys(json).length === 1 ? '{ "extends": "@definitelytyped/dtslint/dt.json" }' : JSON.stringify(json, undefined, 4);
-    fs.writeFileSync(target, text + "\n", "utf-8");
+    const text =
+        Object.keys(json).length === 1 ? '{ "extends": "dtslint/dt.json" }' : JSON.stringify(json, undefined, 4);
```

Comment:
I don't think this extends config should be changing.

Reply:
- [x] Thank you! I've undone that change.